crates/hotshot/src/lib.rs

```diff
@@ -256,6 +261,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> SystemContext<TYPES, I> {
 
         let inner: Arc<SystemContextInner<TYPES, I>> = Arc::new(SystemContextInner {
             id: nonce,
+            #[cfg(feature = "hotshot-testing")]
```

Comment:
@shenkeyao Can we make an issue to delete this field?  It looks like we don't use it anymore.

Reply:
While you're at it... there's another field in this struct called `_metrics` that also appears not to be in use. Maybe we could delete it, too. 🙏

Reply:
Sure! Issue created: https://github.com/EspressoSystems/HotShot/issues/2406.